flight/targets/sprf3e/fw/pios_config.h

```diff
@@ -0,0 +1,96 @@
+/**
+ ******************************************************************************
+ * @addtogroup Targets Target Boards
+ * @{
+ * @addtogroup spf3e SP Racing F3E support files
+ * @{
+ *
+ * @file       sprf3e/board-info/pios_config.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2015
+ * @author     dRonin, http://dRonin.org/, Copyright (C) 2016
+ * @brief      Board specific options that modify PiOS capabilities
+ * @see        The GNU Public License (GPL) Version 3
+ *
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, see <http://www.gnu.org/licenses/>
+ */
+
+#ifndef PIOS_CONFIG_H
+#define PIOS_CONFIG_H
+
+#include <pios_flight_config.h>
+
+/* Major features */
+
+/* Enable/Disable PiOS Modules */
+#define PIOS_INCLUDE_DMA_CB_SUBSCRIBING_FUNCTION
+#define PIOS_INCLUDE_I2C
+#define PIOS_INCLUDE_SPI
+#define WDG_STATS_DIAGNOSTICS
+
+/* Select the sensors to include */
+#define PIOS_INCLUDE_MPU9250_SPI
+//#define PIOS_MPU6000_ACCEL
+#define PIOS_INCLUDE_BMP280
+#define PIOS_INCLUDE_HMC5883
+#define PIOS_INCLUDE_HMC5983_I2C
+#define PIOS_TOLERATE_MISSING_SENSORS
+
+/* Com systems to include */
+#define PIOS_INCLUDE_COM_TELEM
+//#define PIOS_INCLUDE_TELEMETRY_RF
+//#define PIOS_INCLUDE_COM_FLEXI
+#define PIOS_INCLUDE_MAVLINK
+#define PIOS_INCLUDE_MSP_BRIDGE
+#define PIOS_INCLUDE_SESSION_MANAGEMENT
+#define PIOS_INCLUDE_LIGHTTELEMETRY 
+
+
+/* Supported receiver interfaces */
+#define PIOS_INCLUDE_PWM
+
+
+/* Flags that alter behaviors - mostly to lower resources for CC */
+//??#define PIOS_INCLUDE_INITCALL           /* Include init call structures */
```

Comment:
You can delete this line, it's defined in pios_flight_config.h which you have included already.